GIT_AND_GITHUB.md

```diff
@@ -47,19 +47,30 @@ Use `--no-ff` when you merge to `master`.
 It's okay to just push to `master` if it's a quick fix and you really need
 it out the door. Otherwise, make it a pull request.
 
+### Pull requests
+
 Pull request descriptions should be concise and well written. The merger should
 be able to copy this description straight into the release notes instead of
 figuring out what changed or was fixed.
 
-#### Who merges the pull request?
+Besides a clear description more information could be helpful, for example:
+
+- If is static UI, a screenshot would do. 
+- If is an interaction a gif would help a lot. A good tool to make gifs is [Licecap](http://www.cockos.com/licecap/).
+- If is a bug, steps to reproduce are useful to understand the context.
+
+#### When to merge a pull request?
+
+Before merging a pull request your code has to be reviewed so we can learn from you, 
+point out your silly mistakes and/or post the sufficient amount of gifs.
 
-Core contributors have the final say on what gets merged into their codebase,
-so they get to click the button. That could be you, but if you're working with
-someone else it should probably be them. The important thing is that someone else
-gets to look it over so we can learn from you, point out your silly mistakes and/or
-post the sufficient amount of gifs.
+When the reviewer accepts all the changes in the PR and confirms that the changes are ready 
```

Comment:
I would prefer to have the confirmation open. It can be a gif, a hug, whatever. Using only :+1: gets repetitive and boring.